custom_components/tuya_local/devices/kogan_lx10_vacuum.yaml

```diff
@@ -32,19 +36,19 @@ primary_entity:
         - dps_val: pointing\areaing
           value: pointing_areaing
         - dps_val: totaling
-          value: totaling
+          value: cleaning
         - dps_val: sweep
           value: sweep
         - dps_val: mop
           value: mop
         - dps_val: fault
-          value: fault
+          value: error
         - dps_val: pause
-          value: pause
+          value: paused
         - dps_val: chargring
-          value: charging
+          value: docked
```

Comment:
"charging" is also recognised as a status which translates to a docked state (status and state are separate in HA's vacuum  entity - state has very limited set of fixed states to choose from, status can be anything)